packages/radix-ui-themes/src/components/table.css

```diff
@@ -35,6 +35,14 @@
   padding: var(--table-cell-padding);
   /* Works as min-height */
   height: var(--table-cell-min-height);
+
+  /* Inset with Table */
+  .rt-Inset :where(&:first-child) {
+    padding-left: var(--inset-padding, var(--table-cell-padding));
+  }
+  .rt-Inset :where(&:last-child) {
+    padding-right: var(--inset-padding, var(--table-cell-padding));
+  }
```

Comment:
This was `.rt-Inset > .rt-TableRoot .rt-TableCell:first-child`.
Do we need the direct descendant selector and table root?

Reply:
Again, not having feels more brittle to me (ie. you might have an inset, and then some intermediary elements where you've handled spacing and now the table does its thing too)

Reply:
Decided to keep it this way as it's a bit of rabbit hole with all the possible Inset sides and fringe cases with layout